libgnucash/app-utils/gnc-gsettings.h

```diff
@@ -86,6 +86,16 @@ void gnc_gsettings_set_prefix (const gchar *prefix);
 const gchar *gnc_gsettings_get_prefix (void);
 
 
+/** Block all prefs callbacks, used while preference dialog is loaded.
+ */
+void gnc_gsettings_block_all (const gchar *text);
```

Comment:
Can you give more details of how this was implemented with void ? You can also create a temporary commit with all the changes if you like (even if it doesn't work). That may be easier to figure out what is wrong.